x-pack/plugins/threat_intelligence/public/modules/indicators/components/indicators_flyout/tabs/indicators_flyout_overview/components/block/indicator_block.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiPanel, EuiSpacer, EuiText } from '@elastic/eui';
+import React, { VFC } from 'react';
+import { euiStyled, css } from '@kbn/kibana-react-plugin/common';
+import { Indicator } from '../../../../../../../../../common/types/indicator';
+import { IndicatorFieldValue } from '../../../../../indicator_field_value';
+import { IndicatorFieldLabel } from '../../../../../indicator_field_label';
+import { IndicatorValueActions } from '../../../../../indicator_value_actions';
+
+/**
+ * Show actions wrapper on hover. This is a helper component, limited only to Block
+ */
+const VisibleOnHover = euiStyled.div`
```

Comment:
would you be ok moving this to a `style.ts` file to stay consistent with the rest of the plugin?
If the PR is ready to go, we can make this change later!